tests/basics/fixed_dict.py

```diff
@@ -0,0 +1,48 @@
+# test that fixed dictionaries cannot be modified
+
+try:
+    import uerrno
+except ImportError:
+    print("SKIP")
+    raise SystemExit
+
+# Save a copy of uerrno.errorcode, so we can check later
+# that it hasn't been modified.
+errorcode_copy = uerrno.errorcode.copy()
+
+try:
+    uerrno.errorcode.popitem()
+except TypeError:
+    pass
```

Comment:
Please make this a `print('TypeError')` because otherwise there's no way to tell if an exception was raised or not, and in this case an exception should be raised.  Then you'll need to make a .exp file that has the expected output (because this script can't be checked by CPython).